### DISCUSSION TITLE
Challenge setting up Avail Madara

### DISCUSSION THREAD

**Discussion by Cryptmenic** [availproject/avail#415] [Q&A]
![Avail](https://github.com/availproject/avail/assets/88225567/713d53db-12f5-4a3d-8c02-34cda274a22e)

Anytime I want to run my appchain, I alsways encounter this error. I have reinstalled my VPS over 3 times and always have the same error. Kindly help.

**Comment by lizyld** [comment]
same as mine... it's already gave adress not same as mine wallet it new... I can't see where I can see the orhases

**Comment by Leouarz** [comment]
Closing now that challenge is over
